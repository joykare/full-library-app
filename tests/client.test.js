import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import LibraryApp from "../client/components/libraryApp.js";

describe("Library Component", () => {
  it("snapshot test", () => {
    const wrapper = shallow(<LibraryApp />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  })
})