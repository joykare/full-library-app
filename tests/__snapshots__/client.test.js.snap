// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Library Component snapshot test 1`] = `
<div>
  <div
    className="text-center"
    style={
      Object {
        "paddingBottom": 10,
        "paddingTop": 10,
      }
    }
  >
    <h4>
      Library App 
    </h4>
  </div>
  <hr />
  <div
    className="jumbotron"
    style={
      Object {
        "marginLeft": 100,
        "marginRight": 100,
      }
    }
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className="form-group"
      >
        <label>
          Title:
        </label>
        <input
          className="form-control"
          onBlur={[Function]}
          placeholder="Enter book title"
          type="text"
        />
      </div>
      <div
        className="form-group"
      >
        <label>
          Author:
        </label>
        <input
          className="form-control"
          onBlur={[Function]}
          placeholder="Enter name of book author"
          type="text"
        />
      </div>
      <button
        className="btn btn-primary"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
  <div
    className="row"
    style={
      Object {
        "marginBottom": 20,
      }
    }
  />
</div>
`;
